scenario/scenario_shacl.ttl

```diff
@@ -278,4 +203,88 @@ scenario:TrafficShape a sh:NodeShape ;
+            sh:path scenario:temporaryTrafficObjects ],
+        [ skos:example "10" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of numberTrafficObjects failed!"@en ;
+            sh:name "number traffic objects"@en ;
+            sh:order 0 ;
+            sh:path scenario:numberTrafficObjects ],
+        [ skos:example "myDriver, BestPedestrianController2000" ;
+            sh:datatype xsd:string ;
+            sh:description "Usage of simulation core internal or external controllers."@en ;
+            sh:message "Validation of controllers failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "controllers"@en ;
+            sh:order 5 ;
+            sh:path scenario:controllers ],
+        [ skos:example "8" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic permanent objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of permanentTrafficObjects failed!"@en ;
+            sh:name "permanent traffic objects"@en ;
+            sh:order 2 ;
+            sh:path scenario:permanentTrafficObjects ] ;
+    sh:targetClass scenario:Quantity .
+
+scenario:QualityShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "0.1" ;
+            sh:datatype xsd:float ;
+            sh:description "Accuracy of moving objects (only applicable if scenario is based on real measurments)."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of accuracyObjects failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "accuracy objects"@en ;
+            sh:order 4 ;
+            sh:path scenario:accuracyObjects ],
+        [ skos:example "Static sensor calibration performed at <origin> using system XYZ" ;
+            sh:datatype xsd:string ;
+            sh:description "Description of any calibration steps performed prior to measurement"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of calibration failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "calibration"@en ;
+            sh:order 5 ;
+            sh:path scenario:calibration ] ;
+    sh:targetClass scenario:Quality .
+
+scenario:DataSourceShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "Real World Data" ;
+            sh:datatype xsd:string ;
+            sh:description "Source type"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of source type failed!"@en ;
+            sh:in ("Accident Database" "Real World Data" "Analytical Hazard Based Approach" "Formal Verification" "Operational Design Domain" "Ontology" "Standards, Regulations, Guidelines" "Real-world Deployment and Trials") ;
+            sh:minCount 0 ;
+            sh:name "source type"@en ;
+            sh:order 0 ;
```

Comment:
in der DataSourceShape haben wir zweimal sh:order 0.

Reply:
fixed